app/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,12 @@
     <string name="nav_header_subtitle">android.studio@android.com</string>
     <string name="nav_header_desc">Navigation header</string>
     <string name="action_settings">Settings</string>
+    <!--CONTENT MAIN-->
+    <string name="products_button">Produkty</string>
+    <string name="recipes_button">Przepisy</string>
+    <string name="database_button">Baza wiedzy</string>
+    <string name="logo">Logo of DDApp</string>
+    <!--REDIRECT ACTIVITY-->
+    <string name="redirectMessage">This feature is not yet implemented.</string>
```

Comment:
Piszmy na razie wszystko po polsku - później ew. będziemy to tłumaczyć na inne języki dodając nowe pliki strings.xml

Reply:
To jasne, zapomniałem się i z przyzwyczajenia zacząłem tak :D

Reply:
Nie ma problemu :smile: